docs/operator/configuration_parameters.md

```diff
@@ -532,36 +532,48 @@ and only contains information about the specific RSE's for the Rucio instance.
 Read more about RSEs [here](../started/concepts/rucio_storage_element.md)
 and how to set them up [here](./configuration.md#creating-new-rses).
 
-- **associated_sites**: <!--??--> Separated by commas. Default: `None`.
-- **auto_approve_bytes**: Auto Approve Limit. Example: `500GB`. No default.
+- **associated_sites**: <!--??--> String. Separated by commas. Used for chaining of subscriptions so that transfers to one RSE will also be mirrored to associated_sites. No default.
+- **archive_timeout**: <!--??--> Integer: For transfers with a tape destination, control the number of seconds Rucio will wait for the tape archival of the file to complete. No default.
```

Comment:
```suggestion
- **archive_timeout**: <!--??--> Integer. Only used for transfers with a tape destination. Controls the number of seconds Rucio will wait for the tape archival of the file to complete. No default.
```

Not completely clear why Rucio would need to wait at all, is this the delay before marking a file as completed? What operation does it move onto?

Reply:
This is controlling the timeout for a file to make it to tape, where Rucio wants to be certain the file has been archived to tertiary storage before marking the transfer to the tape RSE as complete.

Reply:
To clarify this a little more accurately, `archive_timeout` is actually passed as a job parameter to FTS3. From the Rucio perspective the DID just stays in REPLICATING. To FTS3 this state is in ARCHIVING, which transitions to FAILED or FINISHED as appropriate.

I guess I can clarify the note on this a little better.